### ISSUE TITLE
[Feature] move_window_x command and numeric prefix in movement mode

### ISSUE THREAD

**Issue by qaptoR** [pit-ray/win-vind#146]
Movement mode is excellent for making adjustments on the fly, but two things are missing

1. normal ability to bind `move_window_*` for more immediate adjustments, and also to work with numeric prefix buffer
eg. 10sj (move 10 increments down)

2. when entering movement mode, enabled to use numeric prefix buffer
eg. 10j (when in window movement mode)

3.  adding `window_hvelocity` and `window_vvelocity`
this is necessary because pixel ratio means that dividing screen horizontally and vertically using `window_hdelta` and `window_vdelta` must be in equal proportion
ie. current `window_velocity` must be lowest common denominator between `window_*delta`

using the numeric prefix buffer for window movement is an absolute must for when `window_*velocity` is set low

**Comment by pit-ray** [comment]
@qaptoR 
Thanks for your contributions.
Is `window_resizer` not enough?
This function allows us to move window with hjkl.

**Comment by qaptoR** [comment]
You make a good point. For my previous comment .1 I think that the `window_resizer` does do a very good job, especially as I use it more and more, I'm not sure I would want to use specific keybinds, as I more than likely have to move the window in multiple directions to it's new position.

However for .2, it is would be extremely useful to move windows with motions, like 4h, or 4k, Especially until .3 is implemented or if it never is.

.3 is important because, for instance if I set `window_hdelta` to 90, but `window_vdelta` to 50, then I must set the current `window_velocity`  to 10 because that is the lowest common factor between them.
So, even thought my monitor is divided into equal 90-pixel segments, I have to press `h` 9 times to move 1 segment (or press and hold for an extended period), or if .2 is implemented then I could type `9h` which is a decent workaround.
Whereas if there were instead 2 separate `window_hvelocity` and `window_vvelocity`, then I can just press `h` 1 time to move 1 90-pixel segement.


So in summary. of my previous comments
1 is not very important at all
3 is somewhat important, but optional
2 is very important for quality of life
